jwst/exp_to_source/tests/data/jwst_nod1_cal.fits

```diff
@@ -199,29 +183,65 @@ slits:
                       forward:
                       - !transform/polynomial-1.1.0
                         coefficients: !core/ndarray-1.0.0
-                          source: 1
+                          data:
```

Comment:
@nden this and similar below makes it seem like we might have missed an `inline_threshold` somewhere? I can look into it.

Reply:
Hm, yes.